core/bentley/src/UtilityTypes.ts

```diff
@@ -93,6 +93,13 @@ export type AsyncFunction = (...args: any) => Promise<any>;
  */
 export type AsyncMethodsOf<T> = { [P in keyof T]: T[P] extends AsyncFunction ? P : never }[keyof T];
 
+/** The subset of `T` including only functions that return a promise, and no other properties
+ * @public
+ */
+export type PickAsyncMethods<T> = {
```

Comment:
can `AsyncMethodsOf` be changed to use `PickAsyncMethods`?

Reply:
yes I think `keyof PickAsyncMethods<T>` would be equivalent.

Reply:
done